src/sections/Docker-Meshery/docker-extension-meshery.js

```diff
@@ -206,6 +229,42 @@ const DockerExtensionMeshery = () => {
                   name="org"
                   required
                 />
+                <div className="accounts">
+                  <label className="form-name">Account(s) to Connect<span className="required-sign">*</span></label>
+                  <p>
+                    Choose between Twitter, Google, LinkedIn, and GitHub, provide the username/handle of your user account for your preferred identity provider. Selected beta program participants will receive a free Meshery Cloud account and have full access MeshMap enabled for each of the following user accounts that you provide. Please provide at least one account.
+                  </p>
+                  {validateAccounts && <p style={{ margin: "0px", color: "red" }}>{errorAccounts}</p>}
+                  <div className="accounts_group">
+                    <label htmlFor="google" className="form-name">Google</label>
+                    <Field type="text" placeholder="my-address@gmail.com" className="text-field" id="google" name="google" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" />
+                    <label htmlFor="github" className="form-name">GitHub</label>
+                    <Field type="url" placeholder="https://github.com/" className="text-field" id="github" name="github" pattern="http(s?)(:\/\/)((www.)?)github.com(\/)([a-zA-z0-9\-_]+)" />
+                    <label htmlFor="twitter" className="form-name">Twitter</label>
+                    <Field type="url" placeholder="https://twitter.com/" className="text-field" id="twitter" name="twitter" pattern="http(s?)(:\/\/)((www.)?)twitter.com(\/)([a-zA-z0-9\-_]+)" />
+                    <label htmlFor="linkedin" className="form-name">Linkedin</label>
+                    <Field type="url" placeholder="https://www.linkedin.com/" className="text-field" id="linkedin" name="linkedin" />
+                  </div>
+                </div>
+
+                <label htmlForFor="role" className="form-name">What role best identifies you?<span className="required-sign">*</span></label>
+                <div role="group" className="formRight" aria-labelledby="select">
+                  <Field as="select" name="role">
```

Comment:
For `Select` a custom dropdown arrow was created for the MeshMap form. Please use the same here as well.

Reply:
Sigh. I assumed that the other MeshMap form had these items taken care of, @warunicorn19 @SAEb-ai @Nikhil-Ladha.

I just need this done and don't care who does it. I'll check in when I wake tomorrow, if it's not done, I'll close all related PRs and rework this system.